### DISCUSSION TITLE
如何设置OpenClash的代理模式为GFW模式？

### DISCUSSION THREAD

**Discussion by AlephZHAO** [vernesong/OpenClash#3382] [Q&A] [answered]
在网上搜了很多，都没有大佬分享如何设置OpenClash的代理模式为仅GFW列表模式；所以来到这里发起QA，想问下大神们，如何设置OpenClash为**仅代理GFW列表**模式？十分感谢！

**Comment by ghost** [comment, marked as answer]
Clash没有什么仅GFW列表模式一说。

如果你指的是国外全部走节点流量，国内不走，又不想自己动手大范围改规则文件，可以这样子操作 `插件设置-流量控制-实验性：绕过中国大陆 IP`（如果有用ipv6的话，要去IPv6设置里单独开） `插件设置-大陆白名单订阅-自动更新|检查并更新` `插件设置-模式设置-代理模式-global全局代理` （`覆写设置-规则设置-仅代理命中规则流量`这个选项我建议完全不懂clash的打开）设置完重新启动一下clash 

clash 的规则文件是可以很细化的那种，一般机场用户用的比较多，因为可以在用 A 节点查 Google 的同时用 B 节点看 YouTube 用 C 节点看 Netflix 
有兴趣建议自己学一下 https://dreamacro.github.io/clash/configuration/rules.html
如果英文看着不太方便可以看 [Meta](https://wiki.metacubex.one/config/rules/geox/) 的但有些功能与原版 clash 不兼容

**Reply by AlephZHAO** [reply]
十分感谢您的回答！
Clash确实是一款有着**强大分流能力**的代理软件，其中一些设置也比其他的代理软件稍复杂一些；我很感兴趣并且正在学习如何使用OC；您的方法我试了一下，**完全没有问题**！再次感谢您！

**Reply by ghost** [reply]
额，发现犯傻了，不好意思，开 `global` 后 `仅代理命中规则流量` 是不生效的，自行注意下bt吧。
一般机场自带规则够用了吧？是不好用吗？
最好代理模式还是能换回 `rule` 或者 `script` 。

**Reply by AlephZHAO** [reply]
好的，我已经配置好代理模式了，改回了`Rule`模式
机场自带的规则确实够用了，不过我看其他的代理软件（如passwall）都有GFWL模式，所以像研究一下OC的GFWL模式；现在我知道了，并没有GFW.
**感谢您的回答！**

**Reply by zxggg566** [reply]
自己写规则呗

 - GEOSITE,gfw,GFW
 - GEOIP,cn,直连
 - MATCH,直连

**Reply by AlephZHAO** [reply]
> 自己写规则呗
> 
> * GEOSITE,gfw,GFW
> * GEOIP,cn,直连
> * MATCH,直连

可行的，感谢大佬

**Reply by MusicLifa** [reply]
> 自己写规则呗
> 
> * GEOSITE,gfw,GFW
> * GEOIP,cn,直连
> * MATCH,直连

可否详细一点，这个添加在哪里？我只想openclash走gfwlist黑名单规则，在哪里添加？
